cli/commands/db/deploy.ts

```diff
@@ -0,0 +1,41 @@
+import { red, green } from 'chalk'
+import auth0 from '../../lib/client'
+import { generateScript, getAllConnections } from '../../lib/utils'
+import { DB_MANIFEST } from '../../manifests'
+import { CONNECTION_NAME } from '../../lib/db-utils'
+
+export default async function run() {
+  try {
+    const connections = await getAllConnections()
+    const dbConnection = connections.find(
+      (conn) => conn.name === CONNECTION_NAME
+    )
+    if (!dbConnection) {
+      throw new Error('Could not find requested Auth0 connection')
+    }
+    if (!dbConnection.id) {
+      // Apparently Auth0 doesn't guarantee that a connection has an id??
+      throw new Error('How the hell did we get a connection without an id')
```

Comment:
This error message is fun, but maybe `Connection is missing ID` is actually going to be more helpful to future devs 😛

Reply:
I'm not actually sure. It's definitely not in the normal [register](https://en.wikipedia.org/wiki/Register_(sociolinguistics)) of programming messages, but I think it conveys the thing well. You could probably rephrase it as "Connection is missing ID (this should never happen)", which I guess I'll do.

Reply:
I mean, I guess I feel like the message should say the thing, but not editorialise too much. E.g. if you got the original error message, it suggests a bit more that there's some weird gremlins in your own code, whereas it might actually be that we haven't understood the upstream API model perfectly, and there are good reasons why it might come back with an undefined ID. I think putting this sort of stuff in comments is pretty helpful, but it seems odd in error messages. Not a big deal either way, but this is sort of the frame I'm coming at it from.